gpii/node_modules/processReporter/configs/gpii.processReporter.config.development.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
It seems to me that perhaps we shouldn't actually use a static processreporter - the matchmaker will depend on the processreporter to tell us when an application is running and when it's not - and expect that value to change after we've run the "start" block of an application. Since a static process reporter will always report the same state, it wont be useable with the full system in practice (or at least it would make the matchmaker extremely frustrated and confused :) )... In terms of integration testing, the same thing holds - we need the processreporter to report something different after we've attempted to start an application (we've been doing this with mocks for most other components)...

I might be missing something vital here, but if not, I think it's safe to remove the files and configs related to the static process reporting and simply use the live reporter - even in development mode